app/services/k8s/usage-metrics.ts

```diff
@@ -1,32 +1,76 @@
 import { PodMetric } from '@kubernetes/client-node';
 import { Cluster } from '@prisma/client';
 import { logger } from '@/core/logging';
-import { normalizeMemory, normalizeCpu, Pod } from '@/helpers/resource-metrics';
-import { getK8sClients } from './core';
+import { normalizeMemory, normalizeCpu, PVC, resourceMetrics } from '@/helpers/resource-metrics';
+import { getK8sClients, queryPrometheus } from './core';
 
-export async function getPodMetrics(licencePlate: string, environment: string, cluster: Cluster) {
+async function getPvcUsage(name: string, namespace: string, cluster: Cluster) {
+  const queries = {
+    usage: `kubelet_volume_stats_used_bytes{persistentvolumeclaim="${name}", namespace="${namespace}"}`,
+    capacity: `kubelet_volume_stats_capacity_bytes{persistentvolumeclaim="${name}", namespace="${namespace}"}`,
+    freeInodes: `kubelet_volume_stats_inodes_free{persistentvolumeclaim="${name}", namespace="${namespace}"}`,
+  };
+  const [usageResult, capacityResult, freeInodesResult] = await Promise.all(
+    Object.values(queries).map((query) => queryPrometheus(query, cluster)),
+  );
+
+  if (usageResult.length && capacityResult.length && freeInodesResult.length) {
+    const usage = parseFloat(usageResult[0].value[1]);
```

Comment:
1. please check whether `usageResult` always has one item in the array; if then please include it into `queryPrometheus` function.
2. use `destruction of array` to make the code more readible.
```sh
const [name, value] = usageResult[0].value;
name === usageResult[0].value[0]
value === usageResult[0].value[1]
```